collectors/channels_collector.go

```diff
@@ -317,6 +340,150 @@ func (c *ChannelsCollector) Collect(ch chan<- prometheus.Metric) {
 			)
 		}
 	}
+
+	// Get all remote policies
+	remotePolicies, err := c.getRemotePolicies(getInfoResp.IdentityPubkey)
+	if err != nil {
+		channelLogger.Error(err)
+		return
+	}
+
+	// Export the inbound fee metric for a series of amounts.
+	var feeAmount btcutil.Amount = 100000
+	for {
+		inboundFee := getInboundFee(feeAmount, remotePolicies, remoteBalances)
+		if inboundFee == nil {
+			break
+		}
+		ch <- prometheus.MustNewConstMetric(
+			c.inboundFee, prometheus.GaugeValue,
+			float64(*inboundFee)/float64(feeAmount),
```

Comment:
Why the division?

Reply:
It is a proportional amount. The total fee percentage paid for that amount.

Reply:
Added comment